podvm/Dockerfile.podvm_builder.centos

```diff
@@ -0,0 +1,59 @@
+# Copyright Confidential Containers Contributors
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Creates a builder container image that should be used to build the Pod VM
+# disk inside a container.
+#
+FROM quay.io/centos/centos:stream8
+
+ARG GO_VERSION="1.18.7"
```

Comment:
kata-containers has recently bumped to required a minimum version of go 1.19.3 to built the runtime, so it is probably worth bumping to that version?

Reply:
Is it ok if I bump up the golang version in a follow-on PR? Need to test the CAA binary once after building with golang-1.19 to ensure things are working fine.

Reply:
Yeah, that's fine. From https://github.com/confidential-containers/cloud-api-adaptor/issues/440 you can see that I'm still a bit confused how the kata-shim is able to be built with go 1.18, but if it still works for now, then addressing it separately makes sense!